src/current/_includes/common/upgrade/disable-auto-finalization.md

```diff
@@ -0,0 +1,83 @@
+{% if page.path contains 'kubernetes' %}
+
+<section class="filter-content" markdown="1" data-scope="operator">
+For clusters managed by the Operator, auto-finalization is disabled and cannot be enabled. A major version upgrade is not complete until it is manually [finalized](#finalize-a-major-version-upgrade-manually).
+</section>
+
+<section class="filter-content" markdown="1" data-scope="manual">
+
+By default, auto-finalization is enabled, and a major-version upgrade is finalized when all nodes have rejoined the cluster using the new `cockroach` binary. This means that by default, a major-version upgrade cannot be rolled back. Instead, you must [restore the cluster to the previous version]({% link {{ page.version.version }}/restoring-backups-across-versions.md %}#support-for-restoring-backups-into-a-newer-version).
+
+To disable auto-finalization:
+
+1. Connect to the cluster using the SQL shell:
+
+    ~~~ shell
+    $ kubectl exec -it cockroachdb-client-secure \
+    -- ./cockroach sql \
+    --certs-dir=/cockroach-certs \
+    --host=cockroachdb-public
+    ~~~
+
+1. Set the [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) `cluster.preserve_downgrade_option` to the cluster's current major version.
```

Comment:
this should be the "auto" setting right? Just because someone is using k8s we don't want to tell them to set the old setting do we?

Reply:
It's not clear to me whether the manual StatefulSet (this tab) supports it. It seems that the Operator doesn't and in fact forces the use of the old flag, and it seems that Helm also doesn't support it. Maybe @udnay can help us confirm this.

Reply:
Actually I think here for Manual, this is my mistake. I think Manual might be the only Kubernetes deployment mechanism that supports the new flag.

Reply:
PTAL at the latest changes, and hopefully we can get details about which of the Kubernetes methods support it.

Reply:
If there is a new setting then most likely the operator and helm chart do not take advantage of it unless there was an explicit bug filed for it